preprocessors/azure.py

```
@@ -25,53 +27,56 @@ def __call__(self):
+        if kwargs is None:
+            kwargs = {}
+        for i, chunk in enumerate(tqdm(self.load_chunks(concept_config), desc='Chunks')):
+            chunk_processed = self.concepts_process_pipeline(chunk, concept_type, concept_config, kwargs)
+            if first:
+                self.save(chunk_processed, concept_config, f'concept.{concept_type}', mode='w', i=i)
+                first = False
+            else:
+                self.save(chunk_processed, concept_config, f'concept.{concept_type}', mode='a', i=i)
 
-    def concepts_process_pipeline(self, concepts, concept_type, cfg):
+    def concepts_process_pipeline(self, concepts, concept_type, cfg, kwargs={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)